scripts/restore.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+BACKUP_PATH=$1
+
+if [ -z "$BACKUP_PATH" ];
+then
+    echo "Usage: $0 /tmp/backup-20220828.archive"
+    exit 1
+fi
+
+# Restore using mongorestore excluding `trailing-trade-logs`
+mongorestore --host=binance-mongo --port=27017 --gzip --archive="$BACKUP_PATH" --drop
```

Comment:
Same as the previous comment.
`--host` and `--port` should be loaded from `.env` file